specification/deviceprovisioningservices/Deviceprovisioningservices.Management/GroupIdInformation.tsp

```diff
@@ -0,0 +1,81 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.Devices;
+/**
+ * The group information for creating a private endpoint on a provisioning service
+ */
+@parentResource(ProvisioningServiceDescription)
+model GroupIdInformation
```

Comment:
>GroupIdInformation [](http://example.com/codeflow?start=6&length=18)

Why not call it (and this file) PrivateLinkResource?

Reply:
Similar to Johan's comment below. We will handle later.